### PR TITLE
New possible value for `cider-prompt-save-file-on-load'

### PR DESCRIPTION
**Pull request by Malabarba**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -429,6 +429,9 @@ Buffer name will look like *cider-repl project-name:port*.
   the buffer being loaded, if it's modified:
 
 ```el
+;; Don't prompt nor save
+(setq cider-prompt-save-file-on-load nil)
```
Comment (bbatsov) [inline README.md]: nil -> noconfirm

**Comment by bbatsov** [comment]
Not sure about the name of the config value (sounds a bit confusing to me), but that's definitely a good idea.

**Comment by Malabarba** [comment]
What other value would be better? `save-without-confirming`? `always-save`?

**Comment by bbatsov** [comment]
`always-save` sounds better to me. Others are welcome to voice their opinions as well.

**Pull request merged**